_tour/regular-expression-patterns.md

```diff
@@ -58,3 +58,35 @@ key: margin value: 0
 key: height value: 108px
 key: width value: 100
 ```
+
+Moreover, you can combine Pattern Matching and Regular Expression Patterns:
+
+```scala mdoc
+def saveContactInfomation(contact: String): Unit = {
+  import scala.util.matching.Regex
+
+  val emailPattern: Regex = """^(\w+)@(\w+(.\w+)+)$""".r
+  val phonePattern: Regex = """^(\d{3}-\d{3}-\d{4})$""".r
```

Comment:
What do you think of reusing the example introduced above, `keyValPattern`, instead of introducing a new one?